src/wp-admin/includes/dashboard.php

```diff
@@ -1113,7 +1117,8 @@ function wp_dashboard_events_news() {
 		<?php
 			printf(
 				'<a href="%1$s" target="_blank">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a>',
-				'https://www.classicpress.net/blog/',
+				/* translators: If a Rosetta site exists (e.g. https://es.wordpress.org/news/), then use that. Otherwise, leave untranslated. */
```

Comment:
This overrides previous PR #248. You'll probably need to fetch the latest changes first.

Reply:
Rebase done.